docs/README.md

```diff
@@ -5,15 +5,16 @@ toc_order: 1
 
 # What is Kontena?
 
-[Kontena](http://www.kontena.io) is an open source project for orchestrating and running containerized workloads on a cluster. The Kontena system is comprised of a number of **Kontena Nodes** (meaning servers or virtual machines that run containerized workloads) and a **Kontena Master** (which controls and monitors the Nodes).
+[Kontena](http://www.kontena.io) is an open source project for orchestrating and running containerized workloads on a cluster using multiple containers. The Kontena system is comprised of a number of **Kontena Nodes** (meaning servers or virtual machines that run containerized workloads) and a **Kontena Master** (which controls and monitors the Nodes).
```

Comment:
Do you mean `across a cluster of multiple machines` or `running applications consisting of multiple containers on a cluster`?